precompiles/staking/types.go

```diff
@@ -20,6 +21,11 @@ import (
 	cmn "github.com/evmos/evmos/v15/precompiles/common"
 )
 
+const (
+	DoNotModifyCommissionRate    = -1 // constant used in flags to indicate that commission rate field should not be updated
+	DoNotModifyMinSelfDelegation = -1 // constant used in flags to indicate that min self delegation field should not be updated
+)
+
```

Comment:
Can you add these sentinel values also to the Solidity interface as consts ?

Reply:
Sure!